x-pack/solutions/security/packages/kbn-securitysolution-exceptions-common/api/create_exception_list/create_exception_list.schema.yaml

```diff
@@ -20,6 +20,14 @@ paths:
           application/json:
             schema:
               type: object
```

Comment:
when I tried to execute example, I got a following error

<img width="1143" alt="Screenshot 2025-01-08 at 11 15 03" src="https://github.com/user-attachments/assets/1ffc0dd1-d280-4ad3-b347-9c3dc20d259a" />


Is part of this also to verify APIs work correctly?

Reply:
This is a great catch. I'll follow up on this with the docs team. cc @jmikell821

Reply:
This is not a part of our Bump docs it seems. So we're ok here.